pkg/util/object_meta.go

```diff
@@ -16,19 +16,14 @@ package util
 // Utilities here are used for modifying the ObjectMeta field for k8s object
 
 // CombineStringMap merges maps to the left
-func CombineStringMap(a map[string]string, maps ...map[string]string) map[string]string {
```

Comment:
Should we make sure that the values in the map `a` should not be overridden by the values in the `maps`?

Reply:
changed to keep the first value in https://github.com/pingcap/tidb-operator/pull/4029/commits/437715ec914f7e74a236835467309e9470dd2d0b